.github/labeler.yml

```diff
@@ -0,0 +1,46 @@
+"area:help centre":
+  - changed-files:
+      - any-glob-to-any-file:
+          - wiki/Help_centre/**
+
+"area:legal":
+  - changed-files:
+      - any-glob-to-any-file:
+          - wiki/Legal/**
+
+"area:meta":
+  - changed-files:
+      - any-glob-to-any-file:
+          - "*"
+          - .github/**
+          - meta/**
+          - news/.markdownlint.json
+          - news/.remarkrc.js
+          - wiki/Tournaments/README.md
+          - wiki/Tournaments/TEMPLATE.md
+          - wiki/Tournaments/TEMPLATE-SERIES.md
+
+"area:news":
+  - changed-files:
+      - any-glob-to-any-file:
+          - news/**/*.md
```

Comment:
why does this one get `.md` when others dont?

also probably include the news images directory

Reply:
Because the folder has config files of formatting, although it can be deleted if not necessary